app/src/main/java/com/epfl/drawyourpath/mainpage/fragments/FriendsFragment.kt

```diff
@@ -46,18 +44,31 @@ class FriendsFragment(private val database: Database) : Fragment(R.layout.fragme
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        val database: Database = this.database
```

Comment:
Just a question, is this not used in the handleUsernameSearch ?

Reply:
It is but I added it as an argument to the constructor so it simplifies the testing process(aka when you have to use mock database instead of the firebase)
class FriendsFragment(private val database: Database) : Fragment(R.layout.fragment_friends).
I don't remember specifically what that line was for but when searching the database from the argument above is used.